src/main/features/claim/routes/check-and-send.ts

```diff
@@ -68,7 +68,7 @@ function deserializerFunction (value: any): StatementOfTruth | QualifiedStatemen
 }
 
 function getStatementOfTruthClassFor (draft: Draft<DraftClaim>): { new(): StatementOfTruth | QualifiedStatementOfTruth } {
-  if (draft.document.claimant.partyDetails.isBusiness()) {
+  if (draft.document.claimant.partyDetails.isBusiness() || draft.document.claimant.partyDetails.isSoleTrader()) {
```

Comment:
just a suggestion - we doing this 3 times here - maybe add `isBusinessOrSoleTrader` to partyDetails?

Reply:
I have done as suggested